### DISCUSSION TITLE
آدرس اشتباه صفحه درگاه ملی

### DISCUSSION THREAD

**Discussion by SaeedNoori77** [Sina-Soltani/Parbad#362] [Q&A]
در استفاده از درگاه بانک ملی، PaymentPageUrl اشتباه بوده و آدرسی که برای ریدایرکت داده میشود، به شکل زیر است.

https://sandbox.banktest.ir/melli/sadad.shaparak.ir/VPG/Purchase/Index?token={token}

در آدرس بالا "Index/" اضافه است و  با خطای 404 مواجه میشود. آدرس صحیح باید بصورت زیر باشد

https://sandbox.banktest.ir/melli/sadad.shaparak.ir/VPG/Purchase?token={token}

**Comment by Sina-Soltani** [comment]
سپاس برای اطلاع رسانی.
در نسخه جدید این مشکل برطرف خواهد شد.

**Reply by Sina-Soltani** [reply]
لطفا نسخه جدید رو تست کنید و نتیجه رو اطلاع بدید
https://www.nuget.org/packages/Parbad/3.9.1-beta1

**Reply by SaeedNoori77** [reply]
مشکل رفع شده
تشکر

**Comment by SaeedNoori77** [comment]
سلام وقتتون بخیر
برای این مسئله نسخه عادی منتشر نمیکنین؟

**Reply by Sina-Soltani** [reply]
درود
امروز منتشر میکنم

**Reply by Sina-Soltani** [reply]
نسخه عادی رو میتونید نصب کنید
https://www.nuget.org/packages/Parbad/3.9.1

**Reply by SaeedNoori77** [reply]
ممنون
